x/staging/subspaces/keeper/keeper.go

```diff
@@ -201,7 +201,7 @@ func (k Keeper) UnbanUserInSubspace(ctx sdk.Context, subspaceID, user, admin str
 
 	// Check if the user is already banned inside the subspace
 	if !k.IsBanned(ctx, subspaceID, user) {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "the user is not blocked inside the subspace")
+		return sdkerrors.Wrapf(types.ErrNotBannedUserInSubspace, user)
```

Comment:
I think we can use the more generic `ErrUserNotFound` error (see above):
```suggestion
		return sdkerrors.Wrapf(types.ErrUserNotFound, "%s is not blocked inside the subspace", user)
```